.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Testing
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test:
+    name: Test with Ruby-${{ matrix.ruby }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.0'
+        # TODO: Add jruby if something like allow_failures will be implemented on Actions.
+        ruby: [2.3, 2.4, 2.5, 2.6, 2.7, '3.0', head]
+    env:
+      GITHUB_WORKSPACE: ${{ github.workspace }}
```

Comment:
I think this one is already set: https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

Reply:
Good catch, thanks.